cut-n-paste/gimpcellrenderertoggle/gimpcellrenderertoggle.c

```diff
@@ -61,7 +61,11 @@ static void gimp_cell_renderer_toggle_set_property (GObject         *object,
                                                     GParamSpec      *pspec);
 static void gimp_cell_renderer_toggle_get_size     (GtkCellRenderer *cell,
                                                     GtkWidget       *widget,
+#if GTK_CHECK_VERSION (3, 0, 0)
+                                                    const GdkRectangle *rectangle,
+#else
                                                     GdkRectangle    *rectangle,
```

Comment:
Great, now I get warnings with GTK+2... stupid undocumented API changes. You're right, let's leave it as is.